frontend/lib/styles/components/tooltip.sass

```diff
@@ -39,6 +39,9 @@
   &__key, &__value
     font-size: $font-xs
     margin: 0
+    a, a:visited, a:active
+      color: #0000EE
```

Comment:
What do you think -- would `$col-blue-gray-dark` fit better into the overall design?